.changeset/export_named_type_parser.md

```diff
@@ -0,0 +1,13 @@
+---
+biome_js_parser: patch
```

Comment:
You must add the `cli` package too. Please read the contribution guide regarding the changesets

Reply:
done, thanks for the reminder.